nft_ingester/src/program_transformers/bubblegum/mint_v1.rs

```diff
@@ -154,15 +156,21 @@ where
                     Some(delegate.to_bytes().to_vec())
                 };
                 upsert_asset_with_owner_and_delegate_info(
-                    txn,
+                    &multi_txn,
                     id_bytes.to_vec(),
                     owner.to_bytes().to_vec(),
                     delegate,
                     seq as i64,
                 )
                 .await?;
 
-                upsert_asset_with_seq(txn, id_bytes.to_vec(), seq as i64).await?;
+                upsert_asset_with_seq(&multi_txn, id_bytes.to_vec(), seq as i64).await?;
+
+                upsert_asset_with_update_metadata_seq(&multi_txn, id_bytes.to_vec(), seq as i64)
+                    .await?;
+
+                // Close out transaction and relinqish the lock.
+                multi_txn.commit().await?;
```

Comment:
Why not include the other updates below?

Reply:
I didn't want to lock all the different tables at first.  Starting with `asset` seemed like a good incremental improvement since it had the most upserts.

Reply:
Incorporated all these changes into https://github.com/metaplex-foundation/digital-asset-rpc-infrastructure/pull/134